src/MentionWrapper.js

```diff
@@ -148,10 +155,10 @@ class MentionWrapper extends Component {
   };
 
   selectItem = active => e => {
-    const { options, triggerIdx } = this.state;
-    const preMention = this.ref.value.substr(0, triggerIdx);
+    const { activeTrigger, options } = this.state;
+    const preMention = this.ref.value.substr(0, this.ref.value.lastIndexOf(activeTrigger));
```

Comment:
it looks like itused to pass along the index because we calculate it in maybe mention, its probably faster, can we continue to pass the triggerIdx in addition to the activeTrigger?